web_ui/package.json

```diff
@@ -82,33 +82,31 @@
     "zipson": "^0.2.12"
   },
   "scripts": {
-    "start": "rsbuild dev --environment dev",
-    "preview": "rsbuild preview",
     "build": "rsbuild build --environment geti --environment admin",
-    "build:all-routes": "rsbuild build --environment dev",
-    "build:geti": "rsbuild build --environment geti",
     "build:admin": "rsbuild build --environment admin",
+    "build:all-routes": "rsbuild build --environment dev",
     "build:analyze": "RSDOCTOR=true npm run build",
+    "build:geti": "rsbuild build --environment geti",
     "build:rest-openapi-spec": "npx @redocly/cli bundle --dereferenced ../docs/rest_api/openapi.yaml -o public/openapi-spec.json",
     "build:server": "npm run build:server-schema && npm run build:server-combine && npm run build:server-types",
     "build:server-combine": "ts-node ./src/core/server/combine.mjs",
     "build:server-schema": "swagger-cli bundle -o src/core/server/generated/api-spec.json ../docs/rest_api/openapi_internal.yaml -r",
     "build:server-types": "openapi-typescript src/core/server/generated/api-spec.json --output src/core/server/generated/schemas.ts",
-    "test": "jest --config jest.config.js --watchAll=false",
-    "test:coverage": "npm test -- --coverage",
-    "test:watch": "jest --config jest.config.js --watch",
+    "cyclic-deps-check": "npx madge --circular src/**/*.ts* tests/**/*.ts*",
+    "find-deadcode": "npx knip",
+    "format": "prettier --write --cache \"./src/**/*.{js,ts,jsx,tsx,css,scss,sass,mjs}\" \"tests/**/*.ts*\"",
+    "format:check": "prettier --check --cache \"./src/**/*.{js,ts,jsx,tsx,css,scss,sass,mjs}\" \"tests/**/*.ts*\"",
+    "preview": "rsbuild preview",
+    "start": "rsbuild dev --environment dev",
     "test:component": "npx playwright test --config=playwright.config.ts",
     "test:component:ui": "npm run test:component -- --ui",
+    "test:coverage": "npm test -- --coverage",
     "test:e2e": "npx playwright test --config=playwright.config.ts --project=e2e",
+    "test:eslint": "eslint \"src/**/*.ts*\" \"tests/**/*.ts*\" --max-warnings 0",
```

Comment:
why add "test" prefix on eslint and eslint fix?

Reply:
My main thought here is to group together all the tools we use for quality control. Another idea I was to change thsi to `test:lint` instead of explicitly mentioning eslint. Similarly to how we use `test:unit` and `test:component`, not `test:jest` or `test:playwright` but I'm not sure. What do you think?

Reply:
yeah i like the "lint" idea, but for me "test" means something test related, which is not 100% the case. I would suggest simply "lint"